.github/actions/publish-release/action.yml

```diff
@@ -119,34 +124,49 @@ runs:
         fi
 
     - name: '🛠️ Build and Prepare Packages'
+      if: "${{ inputs.use-bundle-release == 'false' }}"
       working-directory: '${{ inputs.working-directory }}'
       shell: 'bash'
       run: |
         npm run build:packages
         npm run prepare:package
 
     - name: '🎁 Bundle'
+      if: "${{ inputs.use-bundle-release == 'true' }}"
       working-directory: '${{ inputs.working-directory }}'
       shell: 'bash'
       run: |
         npm run bundle
 
     # TODO: Refactor this github specific publishing script to be generalized based upon inputs.
     - name: '📦 Prepare for GitHub release'
-      if: "inputs.npm-registry-url == 'https://npm.pkg.github.com/'"
+      if: "inputs.npm-registry-url == 'https://npm.pkg.github.com/' && inputs.use-bundle-release == 'true'"
       working-directory: '${{ inputs.working-directory }}'
       shell: 'bash'
       run: |
         node ${{ github.workspace }}/scripts/prepare-github-release.js
 
+    - name: '📦 Publish Root Package'
+      if: "${{ inputs.use-bundle-release == 'true' }}"
+      working-directory: '${{ inputs.working-directory }}'
+      env:
+        NODE_AUTH_TOKEN: '${{ inputs.github-token }}'
+      shell: 'bash'
+      run: |
+        npm publish \
```

Comment:
You need to sync this CL, I added a couple additional commands to this to tidy up the "false" tag.